_data/cryptocurrencies.yml

```
@@ -379,6 +379,13 @@ websites:
       - totp
     doc: https://support.poloniex.circle.com/hc/en-us/articles/360025399352
 
+  - name: Remitano
+    url: https://remitano.com/
+    img: remitano.png
+    tfa:
+      - totp
+    doc: https://support.remitano.com/en/articles/781044-how-to-protect-my-account-or-enable-two-factor
```

```suggestion
    doc: https://support.remitano.com/en/articles/781044
```
There's no need for the slack in the documentation URL.

---

Ok, I edited accordingly.